.github/workflows/ci.yml

```diff
@@ -0,0 +1,41 @@
+name: Continuous Integration
+
+on:
```

Comment:
Might also want to add a `schedule` for CRON and `workflow_dispatch` for manual runs &mdash; we've found it handy. :)

c.f. https://github.com/scikit-hep/pyhf/blob/70f100c56588e4119eca5d00ddb101911a4cd50e/.github/workflows/ci.yml#L3-L9

Reply:
m...m... the what is the use-case for `schedule` trigger?

I believe we had this conversation once. Is it because of the potential broken dependencies that the `>=` usage could produce?

Reply:
Rerunning your CI on a nightly or weekly or whatever you want basis is helpful as dependencies can always break as they update themselves. It is good to know as a heads up that your tests now fail because something else changed even if you haven't changed your codebase.

> Is it because of the potential broken dependencies that the `>=` usage could produce?

I wouldn't say if it because of the use of `>=` only, as we've definitley seen that some projects take SemVer less seriously than others. Unless you're installing with a lock file (ala Poetry or pip-tools) that locks down to the hash level there's no way to be 100% sure that the dependencies of your dependencies haven't changed and so forth.

Having a CRON job just run and check these things for you empirically has at least let me feel more confident that `pyhf` hasn't silently broken.

Reply:
Scheduled trigger added in commit 7fc4b0f95b525d662abde931904cdb29da7f9d4b